mcp/cloudflare-worker/worker.ts

```
@@ -0,0 +1,522 @@
+interface Env {
+    // You can add environment variables here if needed
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)